UndertaleModTool/MainWindow.xaml.cs

```diff
@@ -159,8 +327,13 @@ public MainWindow()
         {
             InitializeComponent();
             this.DataContext = this;
-            ChangeSelection(Highlighted = new DescriptionView("Welcome to UndertaleModTool!", "Open data.win file to get started, then double click on the items on the left to view them"));
-            SelectionHistory.Clear();
+
+            OpenInNewTab(new DescriptionView("Welcome to UndertaleModTool!", "Open data.win file to get started, then double click on the items on the left to view them"), "Welcome!");
+
+            ((Tab)CurrentTab).AutoClose = true;
+            ((Tab)CurrentTab).SelectionHistory.Clear();
+
+            ((TabControl)this.FindName("TabController")).SelectedIndex = 0;
```

Comment:
Probably, `TabController` should be accessible (`TabController.SelectedIndex = 0;`), try to replace it.